web/blog/tenzir-platform-v1.4/index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Tenzir Platform v1.4: Platform Alerts"
```

Comment:
Now that the title is changed you likely need to specify the slug exactly, otherwise the links in the release will be hard to predict.